src/main/java/com/maarten551/tictactoe_backend/logic/GameSessionContainer.java

```diff
@@ -0,0 +1,178 @@
+package com.maarten551.tictactoe_backend.logic;
+
+import com.maarten551.tictactoe_backend.exception.*;
+import com.maarten551.tictactoe_backend.model.FieldCell;
+import com.maarten551.tictactoe_backend.model.GameSession;
+import com.maarten551.tictactoe_backend.model.Lobby;
+import com.maarten551.tictactoe_backend.model.Player;
+import org.springframework.stereotype.Service;
+
+import java.awt.*;
+import java.util.List;
+import java.util.*;
+import java.util.stream.IntStream;
+
+@Service
+public class GameSessionContainer {
+    private final int neededAmountOfFieldCellsInARow = 3;
```

Comment:
Private final must be in CAPITAL letters.. (just like public final)

Reply:
Hey, every company handles this differently. So to be fair this isn't a real problem.

Reply:
CaPS